tests/testsuite/artifact_dep.rs

```diff
@@ -673,7 +703,22 @@ fn build_script_with_bin_artifact_and_lib_false() {
     p.cargo("build -Z bindeps")
         .masquerade_as_nightly_cargo(&["bindeps"])
         .with_status(101)
-        .with_stderr_does_not_contain("[..]sentinel[..]")
+        // This should not contain "sentinel"
+        .with_stderr_data(str![[r#"
+[LOCKING] 2 packages to latest compatible versions
+[COMPILING] bar v0.5.0 ([ROOT]/foo/bar)
+[COMPILING] foo v0.0.0 ([ROOT]/foo)
+error[E0433]: failed to resolve: use of undeclared crate or module `bar`
+ --> build.rs:3:16
+  |
+3 |                bar::doit()
+  |                ^^^ use of undeclared crate or module `bar`
```

Comment:
Also, pulling in rustc errors is not idea

Reply:
Switched to `...`